Procfile

```diff
@@ -0,0 +1,2 @@
+server: rails server -p 3000
+webpacker-assets: bin/webpack-dev-server
```

Comment:
what is this procfile for?

Reply:
It is used by foreman to run both rails server and webpack-dev-server (needed to compile just in time assets)